easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.2-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('libpng', '1.6.34'),
+    ('freetype', '2.9'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
+]
+
+prebuildopts = "export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$EBROOTFREETYPE/include/ && "
```

Comment:
@RvDijk This probably needs to be `...include/freetype2`, and we usually prepend rather than append.

And since it's only relevant for `matplotlib` itself, specify it only for the `matplotlib` extension below?

```python
(name, version, {
    'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
    'checksums': ['725a3f12739d133adfa381e1b33bd70c6f64db453bfc536e148824816e568894'],
    'prebuildopts': "export CPLUS_INCLUDE_PATH=$EBROOTFREETYPE/include/freetype2:${CPLUS_INCLUDE_PATH} && "
}),